Dockerfile

```diff
@@ -54,7 +54,7 @@ ENV APP_USER=node
 WORKDIR $APP_HOME
 
 RUN cd modules/admin-ui \
-	&& REACT_APP_ARRANGER_ADMIN_ROOT=/admin/graphql npm run build
+	&& npm run build
```

Comment:
Env variable being set at runtime, whenever available, else we can better set it in a `.env` than hardwiring them while building.